.env.local.sample

```diff
@@ -1,3 +1,4 @@
 NEXT_PUBLIC_REALM_NAME=<get this from the realm you create at auth.tinajs.dev>
 NEXT_PUBLIC_TINA_CLIENT_ID=<get this from the app you create at auth.tinajs.dev>
 NEXT_PUBLIC_REDIRECT_URI=http://localhost:3000/admin
+NEXT_PUBLIC_USE_LOCAL_CLIENT=1
```

Comment:
I've always found that confusing since it's a string so "false" is truthy. At least with 1 and 0 you're not tempted to do a Boolean check on it

Reply:
On our end, can we do `const useLocal = !(NEXT_PUBLIC_USE_LOCAL_CLIENT  == 'false') `

Or if we're concerned about messing up the truthyness on our end, maybe instead change it to something like
`NEXT_PUBLIC_AUTH_TYPE='none'` // default 'cloud'

Reply:
"1" as a boolean flag seems like the most conventional approach - Vercel does something similar for some of their env vars https://vercel.com/docs/environment-variables#system-environment-variables. `NEXT_PUBLIC_AUTH_TYPE` doesn't quite describe it because it's also determining where to point the graphql requests.

Is an enum like `cloud` and `none` easier to remember than 1 vs 0?

Reply:
ah okay, Vercel using that convention definitely increases my confidence in it. In that case, I'm good with leaving it 👍